composer.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "burdamagazinorg/module-hm_newsletter",
```

Comment:
skip the module- prefix for the package name, use "burdamagazinorg/hm_newsletter" instead